internal/worker/dbaccessor/worker.go

```diff
@@ -353,21 +328,20 @@ func (w *dbWorker) loop() (err error) {
 
 			req.done <- nil
 
-		case apiDetails := <-w.apiServerChanges:
-			if err := w.processAPIServerChange(apiDetails); err != nil {
+		case <-firstCheck:
```

Comment:
This is not guaranteed to be run first.

> A select blocks until one of its cases can run, then it executes that case. 
> It chooses one at random if multiple are ready.

If we do want to ensure it's done first, we should add it above the for loop. If we don't care the order, then can we add a comment?

The worst case is that you'll call handle cluster twice.

Reply:
It's an analog for the old behaviour of _requesting_ the `peergrouper` to send you the current API details. It doesn't need to run first, but it will be the first thing to run if we don't have a started node.

Reply:
I changed it to a hard execution prior to the loop.